flow/designs/sky130hd_fakestack/ariane136/config_hier.mk

```diff
@@ -0,0 +1,22 @@
+include $(dir $(DESIGN_CONFIG))/config.mk
+
+export FLOW_VARIANT = hier
+export SYNTH_HIERARCHICAL = 1
+export MAX_UNGROUP_SIZE = 10000
+export RTLMP_FLOW = True
+
+export SDC_FILE      = ./designs/$(PLATFORM)/$(DESIGN_NICKNAME)/constraint.sdc
```

Comment:
Is there a reason to bring this up with TMP instead of RTL-MP as the default?

Reply:
no reason -- made RTLMP the default flow and deleted the TMP flow.